exercises/roman-numerals/introduction.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Today, most of the world use Arabic numerals.
```

Comment:
```suggestion
Today, most of the world uses Arabic numerals.
```

Reply:
I think it's use because numerals is plural, rather than being a singular proper noun.

Reply:
The grammatical number of 'numerals' has nothing to do with it. Rather, it is only the grammatical number of 'most of the world' that matters. That is the subject of the sentence after all. The 'Arabic numerals' are the subject.

Reply:
FWIW, from Googling, apparently people disagree on this. US tend to treat it as singular, UK as plural. Which is why I probably see it that way being British :)

Reply:
Yeah, I can get into it both ways.

The proportion however is definitely singular.

The UK/US division is mostly artificial, exaggerated by computers. Many variations are current on both sides of the pond.

Reply:
“Many cultures across the world today use Arabic numerals” might sidestep the issue.